clients/fides-js/src/lib/tcf.ts

```diff
@@ -5,18 +5,48 @@
  * In the future, this should hold interactions with the CMP API as well as string generation.
  */
 
-import { CmpApi } from "@iabtechlabtcf/cmpapi";
+import { CmpApi, TCData } from "@iabtechlabtcf/cmpapi";
 import { TCModel, TCString, GVL } from "@iabtechlabtcf/core";
 import { makeStub } from "./tcf/stub";
 
 import { EnabledIds } from "./tcf/types";
-import { vendorIsGvl } from "./tcf/vendors";
+import { decodeVendorId, vendorIsAc, vendorIsGvl } from "./tcf/vendors";
 import { PrivacyExperience } from "./consent-types";
+import { FIDES_SEPARATOR } from "./tcf/constants";
+import { FidesEvent } from "./events";
 
+// TCF
 const CMP_ID = 407;
 const CMP_VERSION = 1;
 const FORBIDDEN_LEGITIMATE_INTEREST_PURPOSE_IDS = [1, 3, 4, 5, 6];
 
+// AC
+const AC_SPECIFICATION_VERSION = 1;
```

Comment:
How do we know this is 1?

Reply:
oh right, I meant to revisit this—that's what they had in their docs as an example and I couldn't find anywhere else that had a version. let me ask if anyone else has come across one!

Reply:
okay @NevilleS confirmed we should use `1`!

Reply:
yay! luckily it's much easier than the TC string 😄